hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/details/ArtifactDetailsLayout.java

```diff
@@ -441,7 +431,8 @@ void onEvent(final SoftwareModuleEvent softwareModuleEvent) {
                 }
             });
         }
-        if (softwareModuleEvent.getSoftwareModuleEventType() == SoftwareModuleEventType.ARTIFACTS_CHANGED) {
+        if (softwareModuleEvent.getSoftwareModuleEventType() == SoftwareModuleEventType.ARTIFACTS_CHANGED
```

Comment:
1. Why do we need the `REMOVE_ENTITY`?
2. This lines are duplicated (see line 423)